src/IO/H5/ReadTabulated.hpp

```diff
@@ -0,0 +1,55 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class h5::Tabulated
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <cstdint>
+#include <hdf5.h>
+#include <string>
+#include <vector>
+
+#include "IO/H5/Object.hpp"
+#include "IO/H5/OpenGroup.hpp"
+
+/// \cond
+class Matrix;
+/// \endcond
+
+namespace h5 {
+/*!
+ * \ingroup HDF5Group
+ *
+ * Contains functions to read in rank-1 and rank-3 datasets from an h5 file.
+ */
+class Tabulated : public h5::Object {
+ public:
+  /// \cond HIDDEN_SYMBOLS
+  static std::string extension() { return ""; }
+
+  Tabulated(bool exists, detail::OpenGroup&& group, hid_t location,
+            const std::string& name);
+
+  Tabulated(const Tabulated& /*rhs*/) = delete;
+  Tabulated& operator=(const Tabulated& /*rhs*/) = delete;
+  Tabulated(Tabulated&& /*rhs*/) noexcept = delete;             // NOLINT
+  Tabulated& operator=(Tabulated&& /*rhs*/) noexcept = delete;  // NOLINT
```

Comment:
I think these NOLINTs are no longer necessary.  If they are please indicate which lint we are ignoring.